src/test/kotlin/com/sourcegraph/cody/KeyboardShortcutTest.kt

```diff
@@ -28,11 +28,8 @@ class KeyboardShortcutTest : BasePlatformTestCase() {
             "cody.editCodeAction" to arrayOf("ctrl alt COMMA", "ctrl alt ENTER"),
             "cody.documentCodeAction" to arrayOf("ctrl alt H", "ctrl alt H"),
             "cody.testCodeAction" to arrayOf("ctrl alt G", "ctrl alt G"),
-
-            // Handles cody.inlineEditUndoAction, inlineEditCancelAction, & inlineEditDismissAction:
-            "cody.editCancelOrUndoAction" to arrayOf("ctrl alt BACK_SPACE", "ctrl alt BACK_SPACE"),
             "cody.editShowDiffAction" to arrayOf("ctrl alt D", "ctrl alt K"),
```

Comment:
whoops - looks like a mistake - shouldnt it be `"cody.fixup.codelens.diff"` now?

Reply:
Good catch!